CMP1903_A1_2324/Game.cs

```diff
@@ -17,5 +18,20 @@ internal class Game
 
         //Methods
 
+        Die die1 = new Die();
+        Die die2 = new Die();
+        Die die3 = new Die();
```

Comment:
Should specify private even though implicit.
Could use array rather than 3 declarations.
Private variables also must be in _camelCase, "_dieX".